pkgs/development/libraries/glm/default.nix

```diff
@@ -1,18 +1,29 @@
-{ stdenv, fetchurl, unzip }:
+{ stdenv, fetchurl, unzip, cmake }:
 
 stdenv.mkDerivation rec {
-  name = "glm-0.9.6.1";
+  version = "0.9.8.4";
+  name = "glm-${version}";
 
   src = fetchurl {
-    url = "mirror://sourceforge/project/ogl-math/${name}/${name}.zip";
-    sha256 = "1s1kpf9hpyq6bdf87nhlkxyr2ay0ip9wqicdma9h8yz4vs20r2hs";
+    url = "https://github.com/g-truc/glm/releases/download/${version}/${name}.zip";
+    sha256 = "1c9cflvx0b16qxh3izk6siqldp9q8qlrznk14br3jdyhnr2gbdx9";
   };
 
-  buildInputs = [ unzip ];
+  buildInputs = [ unzip cmake ];
 
   outputs = [ "out" "doc" ];
 
+  phases = [ "unpackPhase" "buildPhase" "installPhase" ];
```

Comment:
Is there a particular reason why it is necessary to explictly set `phases` here?